web/clusters.go

```diff
@@ -0,0 +1,95 @@
+package web
+
+import (
+	"fmt"
+	//"log"
+	"net/http"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+
+	"github.com/trento-project/trento/internal/consul"
+)
+
+const KV_CLUSTERS_PATH string = "trento/clusters"
```

Comment:
ideally this would be a in a location that we can share with the agent (so that front and backend use the same copy of the const)

Reply:
yeah, I suppose we could introduce a new package for everything related to persistence/storage, but I'm not sure we have enough to put there atm!

Reply:
Agree!
I guess this can come in a future implementation with some kind of shared information package